hack/make.sh

```diff
@@ -71,9 +71,9 @@ function install_containerd
 	else
 		echo "Download and install containerd."
 		wget --quiet \
-			https://github.com/containerd/containerd/releases/download/v1.0.3/containerd-1.0.3.linux-amd64.tar.gz -P $TMP
-		tar xf $TMP/containerd-1.0.3.linux-amd64.tar.gz -C $TMP &&
-			cp -f $TMP/bin/* /usr/local/bin/
+			https://github.com/containerd/containerd/releases/download/v1.0.3/containerd-1.0.3.linux-amd64.tar.gz -P "$TMP"
+		tar xf "$TMP/containerd-1.0.3.linux-amd64.tar.gz" -C "$TMP" &&
```

Comment:
I think in the future we need to make this version configurable at the head of this file rather than hard coding here.